packages/tab/src/tab.css

```diff
@@ -17,13 +17,30 @@ governing permissions and limitations under the License.
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    height: auto;
+    height: var(--spectrum-tabs-vertical-item-height, auto) !important;
```

Comment:
Does this need to be important? The height of `tab` is also set in `spectrum-tab.css` inside the `:host` selector.

I think that his selector has a higher specificity than `:host`?

Reply:
This is actually used to over come the `:host([direction=vertical]) ::slotted(*)` selector in `sp-tab-list`: 
https://github.com/adobe/spectrum-web-components/blob/master/packages/tab-list/src/spectrum-tab-list.css#L138-L143